librz/core/cmd.c

```diff
@@ -4635,6 +4635,23 @@ DEFINE_HANDLE_TS_FCN_AND_SYMBOL(arged_command) {
 	if (res == RZ_CMD_STATUS_WRONG_ARGS) {
 		const char *cmdname = rz_cmd_parsed_args_cmd (pr_args);
 		eprintf ("Wrong number of arguments passed to `%s`, see its help with `%s?`\n", cmdname, cmdname);
+		RzCmdDesc *cd = rz_cmd_get_desc (state->core->rcmd, cmdname);
+		if (cd) {
+			char *cmdname_help = rz_str_newf ("%s?", cmdname);
+			if (!cmdname_help) {
+				goto err;
+			}
+			RzCmdParsedArgs *help_pra = rz_cmd_parsed_args_newcmd (cmdname_help);
+			char *help_msg = rz_cmd_get_help (state->core->rcmd, help_pra, true);
```

Comment:
`help_msg` is not checked of its value. so if something goes wrong, it will print `(null)`

Reply:
is `help_pra` dynamically allocated? if yes then it needs to be freed, otherwise probably it should be a `const` in there

Reply:
fixed